pynicotine/plugins/core_commands/__init__.py

```diff
@@ -245,6 +252,25 @@ def leave_command(self, args, room=None, **_unused):
         self.core.chatrooms.remove_room(room)
         return True
 
+    """ Network Filters """
+
+    def ip_address_command(self, args, user=None, **_unused):
+
+        if self.core.network_filter.is_ip_address(args):
+            self.output(self.core.network_filter.get_known_username(args))
+            return
+
+        if args:
+            user = args
+
+        known_ip_address = self.core.network_filter.get_known_ip_address(user) or False
```

Comment:
`or False` is not necessary here. None, empty lists, empty strings etc. are already falsy.

Reply:
I'm tempted to use your new `get_user_ip_addresses()` function instead, it would almost work but there are extraneous parameters in the way as there's no need to modify the filter. I'd want to get all previous_ip_addressess from both filter lists.